changelog.md

```diff
@@ -1,3 +1,8 @@
+## <a name="2.9.3"/> 2.9.3 ##
+
+* Add support for creating geometry collections
```

Comment:
These changes don't appear to be in this release.

Reply:
@kirankumarkolli which are the changes this release included?